addons/website_theme_install/models/theme_models.py

```diff
@@ -41,6 +41,11 @@ def _convert_to_base_model(self, website, **kwargs):
                 # inherit_id not yet created, add to the queue
                 return False
 
+        if inherit._name == 'ir.ui.view' and inherit and inherit.website_id != website:
```

Comment:
You had some question but it got erased from push force:

> 1) Should the condition be inherit.website_id != website ? (it is	
technically possible by tweaking in the backend but would the COW work	
in that case anyway ? (writing on a specific view with a different	
website in the context...)).

Yes, it is stronger
```python 
if inherit and inherit.website_id != website:
```
definitely more accurate than
```python
if inherit and not inherit.website_id:
```
Even if having a specific from another website should not happen, doesnt cost much

> 2) Should I check all the ancestors instead of the direct parent ?	
I.e. is it possible to have a specific hierarchy with non specific	
elements in it ? (in a real case / in a tweaked backend case...).

No definitely not and we don't handle tweaked backend case. I know one could add a view himself in the backend under a specific one but if we start handling that kind of stuff that's the end.

Reply:
I also added `inherit._name == 'ir.ui.view'` as it can't happen with a theme_ir_ui_view imo.

Reply:
When you arrive at that line, it is only possible for `inherit` to be an `ir.ui.view`... :)